src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.jsx

```diff
@@ -270,7 +270,37 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
     userEvent.click(screen.getByText(/continue/i));
 
     expect(screen.queryByRole('alertdialog')).to.not.be.ok;
-    expect(screen.baseElement).to.contain.text('canceled');
+    expect(screen.baseElement).to.contain.text(
+      'You canceled this appointment.',
+    );
+  });
+
+  it('should display who canceled the appointment', async () => {
+    const url = '/va/21cdc6741c00ac67b6cbf6b972d084c1';
+    const canceledAppointment = JSON.parse(JSON.stringify(appointment));
+    canceledAppointment.attributes.vdsAppointments[0].currentStatus =
+      'CANCELLED BY CLINIC';
+
+    mockSingleAppointmentFetch({
+      appointment: canceledAppointment,
+      type: 'va',
+    });
+
+    mockFacilityFetch('vha_442GC', facility);
+
+    const screen = renderWithStoreAndRouter(<AppointmentList />, {
+      initialState,
+      path: url,
+    });
+
+    await waitFor(() => {
+      expect(document.activeElement).to.have.tagName('h1');
+    });
+
+    // NOTE: This 2nd 'await' is needed due to async facilities fetch call!!!
+    expect(screen.baseElement).to.contain.text(
```

Comment:
I think this would need to be something like `expect(await screen.findByText(/Fort Collins VA Clinic canceled this appointment/i).to.exist;`

The facility info is loaded separately from the appointment, so we have to make sure to wait for it to appear.